### PR TITLE
Feat/event display text

### PR DESCRIPTION
**Pull request by UberWaffe**
feat: Actions and conditions as sentences
In the event editor window will now display the actions and conditions as more human readable sentences. (As far as possible, not everything reads like a proper sentence.)

Requires PR #924

### CODE REVIEW

**Comment by UberWaffe** [comment]
Don't review this yet. I'll have to rebase this soon so I can include the actions and conditions from the now merged PR.

File: src/scenario/scenario_events_parameter_data.c [hunk 1]
```diff
@@ -806,3 +806,363 @@ void scenario_events_parameter_data_get_display_string_for_value(parameter_type
...
+    return result_text;
+}
+
+static uint8_t *append_text(uint8_t *text_to_append, uint8_t *result_text, int *maxlength)
+{
+    int text_length = string_length(text_to_append);
+    result_text = string_copy(text_to_append, result_text, *maxlength);
+    *maxlength -= text_length;
+    return result_text;
+}
```
Comment (crudelios) [inline src/scenario/scenario_events_parameter_data.c:824]: Why the different functions? The first one would work fine as a non-`const` is automatically cast to `const`.

Just use `static uint8_t *append_text(const uint8_t *text_to_append, uint8_t *result_text, int *maxlength)`

**Review by crudelios — CHANGES_REQUESTED** [review]
Just a small comment.

File: src/scenario/scenario_events_parameter_data.c (hunk 1 above)
Comment (UberWaffe) [reply src/scenario/scenario_events_parameter_data.c]: I did not know that. Thanks. Updated.

**Comment by UberWaffe** [comment]
@crudelios Updated with your comment.

**Review by crudelios — APPROVED** [review]

**Pull request merged**